frontend/public/components/debug-terminal.tsx

```diff
@@ -21,11 +30,14 @@ const getDebugPod = (debugPodName: string, podToDebug: PodKind, containerName: s
   delete debugPod.metadata.name;
   delete debugPod.metadata.ownerReferences;
   delete debugPod.metadata.labels;
-  debugPod.metadata.generateName = debugPodName;
+  const workloadAnnotations = pickWorkloadAnnotations(debugPod.metadata.annotations);
+  delete debugPod.metadata.annotations;
```

Comment:
`delete` is not needed since you reassign annotations below anyway.

Reply:
Updated.